src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -0,0 +1,25 @@
+package lotto.view
+
+import lotto.domain.Lottery
+import lotto.domain.LottoStatResult
+import lotto.domain.Rank
+
+object ResultView {
+    fun showPurchasedLotto(lottery: Lottery) {
+        println("${lottery.getCount()}개를 구매했습니다.")
+        lottery.getNumberList().forEach {
+            println(it)
+        }
```

Comment:
p3; println을 줄이는 방법으로 다음과 같이 사용해 볼 수 있겠네요.

```suggestion
        println(
            message = lottery.getNumberList()
                .joinToString(separator = "\n"),
        )
```